lib/Db/ObjectEntityMapper.php

```diff
@@ -73,7 +73,7 @@ public function find($identifier): ObjectEntity
 			->from('openregister_objects')
 			->where(
 				$qb->expr()->orX(
-					$qb->expr()->eq('id', $qb->createNamedParameter($identifier, IQueryBuilder::PARAM_INT)),
+					$qb->expr()->eq('id', $qb->createNamedParameter(is_numeric($identifier) ? $identifier : -1, IQueryBuilder::PARAM_INT)),
```

Comment:
What happens if value -1 is used here?

Reply:
The value -1 will never return a result as id starts at 1 and only increases, but because -1 is a valid integer, it will not break the search question.